app/src/main/java/org/openobservatory/ooniprobe/test/test/Signal.java

```diff
@@ -28,6 +30,6 @@ public Signal() {
     @Override public void onEntry(Context c, PreferenceManager pm, @NonNull JsonResult json, Measurement measurement) {
         super.onEntry(c, pm, json, measurement);
         measurement.is_failed = json.test_keys.signal_backend_status.isEmpty();
-        measurement.is_anomaly = json.test_keys.signal_backend_status.equals("blocking");
+        measurement.is_anomaly = json.test_keys.signal_backend_status.equals(BLOCKED);
```

Comment:
✔️

Reply:
However, we have a bug in probe-cli, which is using the wrong string like we did here 🙄